### PR TITLE
migrate /wallets/find-wallet page

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- Fix breadcrumbs issue https://www.notion.so/efdn/BUG-missing-translation-in-breadcrumb-4376c5e5f209473d91f6e58c8b1aface?pvs=4
- Migrate /wallets/find-wallet
- FindWallet components

## Related Issue
https://www.notion.so/efdn/BUG-missing-translation-in-breadcrumb-4376c5e5f209473d91f6e58c8b1aface?pvs=4

### CODE REVIEW

File: src/pages/wallets/find-wallet.tsx [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+          <Subtitle>
+            {t("page-find-wallet-desc-2")}
+          </Subtitle>
+        </Box>
+        <Image
+          w={{ base: "full", sm: "50%" }}
+          src={FindWalletHeroImage}
+          alt=""
+          loading="eager"
+          objectFit="contain"
```
Comment (wackerow) [inline src/pages/wallets/find-wallet.tsx]:
```suggestion
          priority
          style={{ objectFit: "contain" }}
```

File: src/pages/wallets/find-wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+          </OldHeading>
+          <Subtitle>
+            {t("page-find-wallet-description")}
+          </Subtitle>
+          <Subtitle>
+            {t("page-find-wallet-desc-2")}
+          </Subtitle>
+        </Box>
+        <Image
+          w={{ base: "full", sm: "50%" }}
```
Comment (wackerow) [inline src/pages/wallets/find-wallet.tsx]: @pettinarip Can you confirm is this is appropriate here? Given this is a responsive image size, should we be using `sizes` here, paired with the `useBreakpointValue` hook?

Comment (wackerow) [reply src/pages/wallets/find-wallet.tsx]: Not sure what's causing it, but the image here seems slightly larger than production, causing the hero section to be slightly taller than normal

https://github.com/ethereum/ethereum-org-fork/assets/54227730/80c37b5b-9717-4105-b4d3-b1c3c87846a8

File: src/pages/wallets/find-wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,291 @@
+import { useRef, useState } from "react"
+import { shuffle } from "lodash"
+import { GetStaticProps } from "next"
+import { useRouter } from "next/router"
+import { SSRConfig, useTranslation } from "next-i18next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { Box, Drawer, DrawerBody, DrawerCloseButton, DrawerContent, DrawerHeader, DrawerOverlay, Flex, Hide, Show, Text, useDisclosure, useTheme } from "@chakra-ui/react"
```
Comment (wackerow) [inline src/pages/wallets/find-wallet.tsx]: would just run the linter on this

**Review by wackerow — COMMENTED** [review]
Nice job @corwintines! The breadcrumbs appear to be working appropriately now. Left a couple comments but overall this looks good... functioning appropriately on desktop and mobile and don't see any broken strings.

File: src/pages/wallets/find-wallet.tsx (hunk 2 above)
Comment (pettinarip) [reply src/pages/wallets/find-wallet.tsx]: To match what is in prod, we should do 2 things:
1. Create the same node structure that we have in prod (the Gatsby image component renders the image with a wrapper, while Next image renders only the img tag).
2. Crop the image with the same size as we do in the Gatsby query (600px in this case).

What I've been doing recently would be something like this:
```tsx
// wrap the img and set the layout styles in the wrapper
<Center w={{ base: "full", sm: "50%" }}>
  <Image
    src={FindWalletHeroImage}
    // define a image width as we are doing in the Gatsby queries
    width={600}
    alt=""
    priority
    style={{
      objectFit: "contain",
    }}
  />
</Center>
```

**Review by pettinarip — APPROVED** [review]
Nice job @corwintines

File: src/pages/wallets/find-wallet.tsx (hunk 2 above)
Comment (pettinarip) [reply src/pages/wallets/find-wallet.tsx]: Pushed a commit with those changes.

**Review by wackerow — COMMENTED** [review]
Overall looking great.. just spotted a couple small layout changes to the filter selectors. They appear to be text-align centered, with unbounded width (vs fixed width, same for all filter, with ellipsis overflow):

https://github.com/ethereum/ethereum-org-fork/assets/54227730/3d31adb6-943e-4072-a47d-38f3b7626adb

Fine with breaking this off as a separate task

**Comment by corwintines** [comment]
> Overall looking great.. just spotted a couple small layout changes to the filter selectors. They appear to be text-align centered, with unbounded width (vs fixed width, same for all filter, with ellipsis overflow):
> 
>  Screen.Recording.2023-12-13.at.10.51.11.mov 
> Fine with breaking this off as a separate task

Thanks @wackerow. Reverted the version of react-select to what is in the main repo for now. We can look at doing this version update later probably.

**Pull request merged**